src/runtimes/node/utils/zip.ts

```diff
@@ -132,11 +135,29 @@ const createDirectory = async function ({
         return mkdirAndWriteFile(absoluteDestPath, rewrites.get(srcFile) as string)
       }
 
+      const stat = await cachedLstat(cache.lstatCache, srcFile)
+
+      // If the path is a symlink, find the link target and add the link to a
+      // `symlinks` map, which we'll later use to create the symlinks in the
+      // target directory. We can't do that right now because the target path
+      // may not have been copied over yet.
+      if (stat.isSymbolicLink()) {
```

Comment:
We should probably do the same in `createZipArchive`, but starting with `createDirectory` for now.

@ascorbic what happened when you tried to deploy the function to production?

Reply:
According to the test, it works: https://github.com/netlify/zip-it-and-ship-it/pull/1640/commits/07061e053162c76d76bdfd7d41317d47dc0f5e85.

Reply:
Yeah, it worked fine in builds